src/components/checkbox/checkbox.ts

```diff
@@ -55,7 +55,10 @@ export class MCheckbox extends ModulVue {
 
     @Emit('click')
     onClick(event: MouseEvent): void {
-        this.$refs['checkbox']['blur']();
+        // NOTE: this.$refs['checkbox'] is undefined when using Edge
+        if (this.$refs['checkbox']) {
+            this.$refs['checkbox']['blur']();
```

Comment:
1- Je comprend que c'est le comportement actuel et que tu n'as rien changé à ce niveau, mais je vois mal pk on blur le checkbox quand on click dessus... on devrait laisser les focus se faire le naturellement possible... si le standard c'est de focuser le checkbox quand on click dessus, le checkbox devrait focuser
2- (this.$refs.checkbox as HTMLInputElement).blur() :p.  En faisant ça on utilise typescript au lieu de le contourner

Reply:
J'ai changé la syntaxe pour celle que tu as inscrite au point 2.

Reply:
@Mboulianne , je suis d'accord avec toi qu'on devrait gérer le focus et le click separement. voir https://developer.mozilla.org/fr/docs/Web/HTML/Element/Input/checkbox ou 
 encore https://developer.mozilla.org/fr/docs/Web/HTML/Element/Button car on fait la même chose dans le m-button.  Ça me causait des problème au niveau des validations dans le m-form justement,

Par contre je suis d'accord qu'un breaking change est hors scope de cette PR "bugfix" mais il faudrait statuer la dessus et voir ce que l'on fait @jipigi !